ops/docker/ci-builder/Dockerfile

```diff
@@ -54,14 +54,9 @@ RUN apt-get update && \
   pip install slither-analyzer==0.9.3 && \
   go install gotest.tools/gotestsum@latest && \
   curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.48.0 && \
-  curl -fLSs https://raw.githubusercontent.com/CircleCI-Public/circleci-cli/master/install.sh | bash &&
-  chmox +x /usr/local/bin/check-changed
-
-RUN echo "downloading pnpm" && \
-  curl -o /pnpm-install.sh -L https://get.pnpm.io/install.sh && \
-  chmod +x /pnpm-install.sh && \
-  /pnpm-install.sh && \
-  rm /pnpm-install.sh
+  curl -fLSs https://raw.githubusercontent.com/CircleCI-Public/circleci-cli/master/install.sh | bash
```

Comment:
The pnpm bash installer has a bug where it cannot detect the (bash) shell type of the debian baseimage.

Reply:
Noticed this issue in Dockerfile.packages as well and did same fix of using npm i -g pnpm instead